src/components/card/z-card-info/index.tsx

```diff
@@ -65,21 +65,35 @@ export class zCardInfo {
     );
   }
 
+  setExpirationLicenseMessage(type: string) {
+    switch (type) {
```

Comment:
Forse si potrebbe evitare un po' di codice ripetuto così?
`if (
    (type === "online" && this.cardData.onlineLicense.expired)
    || (type === "offline" && this.cardData.offlineLicense.expired)
) {
    return <span class="expired">SCADUTA</span>;
}

if (
    (type === "online" && this.cardData.onlineLicense.expiring)
    || (type === "offline" && this.cardData.offlineLicense.expiring)
) {
    return <span class="expiring">IN SCADENZA</span>;
}`